brainscore_vision/benchmarks/coggan2024_fMRI/test.py

```diff
@@ -0,0 +1,50 @@
+# Created by David Coggan on 2024 06 26
+
+from pytest import approx
+from brainscore_vision import benchmark_registry, load_benchmark
+from brainscore_vision import load_model
+
+
+def test_benchmark_registry():
+    for region in ['V1', 'V2', 'V4', 'IT']:
+        assert f'Coggan2024_fMRI.{region}' in benchmark_registry
```

Comment:
test expressiveness suggestion (need to `import pytest`). This will print the tests with the identifier
```suggestion
@pytest.mark.parametrize(region, ['V1', 'V2', 'V4', 'IT'])
def test_benchmark_registry(region):
    assert f'Coggan2024_fMRI.{region}' in benchmark_registry
```